pkg/services/sqlstore/migrations/ualert/provisioning_permisisons_mig.go

```diff
@@ -0,0 +1,97 @@
+package ualert
+
+import (
+	"fmt"
+	"maps"
+	"slices"
+	"time"
+
+	"xorm.io/xorm"
+
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+)
+
+func ProvisioningPermissionsMigration(mg *migrator.Migrator) {
```

Comment:
We've tested these types of migrations with https://github.com/grafana/grafana/blob/048d1e7c861c419c7b2522429b654cc17a9c903b/pkg/services/sqlstore/migrations/accesscontrol/test/dashbord_permission_migrator_test.go

I think this might be justified in this case as a failed migration here bootloops the instance